libs/damap/src/lib/components/dmp/dmp.component.ts

```diff
@@ -275,4 +283,53 @@ export class DmpComponent implements OnInit, OnDestroy {
   private generateReferenceHash(): string {
     return this.username + (+new Date()).toString(36);
   }
+
+  get emptyProject() {
+    return this.isFormControlEmpty(this.projectStep);
+  }
+
+  get emptyContributor() {
+    return this?.contributorStep?.length === 0;
+  }
+
+  get emptyDocData() {
+    return this?.isFormGroupEmpty(this.docDataStep);
+  }
+
+  get emptyLegalEthical() {
+    return this?.isFormGroupEmpty(this.legalEthicalStep);
+  }
+
+  get emptyExternalStorage() {
+    return this?.externalStorageStep?.length === 0;
+  }
+
+  get emptyRepo() {
+    return this?.repoStep?.length === 0;
+  }
+
+  get emptyReuse() {
+    return this.isFormGroupEmpty(this.reuseStep);
+  }
+
+  get emptyCost() {
+    return this.isFormGroupEmpty(this.costsStep);
+  }
+
+  isFormGroupEmpty(formGroup: UntypedFormGroup): boolean {
```

Comment:
Hey great work, i really love this new feature. I just noticed, that you could use the `summary.service.ts` service instead of checking the form group. This service is currently used in the summary step. This would have the added benefit of being able to differentiate between empty, worked on and done. Steps that are completely done could then be shown as green icons with a checkmark e.g. Sorry for not mentioning this sooner, I did not think about this at all when reviewing the last time 😅

Reply:
Valen, I tested some green colors, but I'm not sure.
Here are the pictures of the tests.

color : --success-container
<img width="947" alt="image" src="https://github.com/user-attachments/assets/cc6c9166-9f53-4651-a03e-4b824c2d68b1">

color: #94C11C <- This green is the same at the logo DMP in the app
<img width="959" alt="image" src="https://github.com/user-attachments/assets/3110c1aa-29ce-4d0b-bd5f-f37c1adaac4c">

Here I think there are many colors for the icons.

In this scenario, when a step is fully completed, the icon is the green check.
If a step is not fully complete but has some info, it is the blue pencil icon.
When a step has no info, the icon is the gray number.

<img width="951" alt="image" src="https://github.com/user-attachments/assets/ecee9177-1e2a-4f3c-b4f1-2ae1a610b38b">
Let me know what do you think about that.


Also I made the changes to use the sumary.service instead of check the form group.

Reply:
Using the tertiary color would also be an option - or you could create a green color palette - would prefer the first option though.

Reply:
Yes, I updated the PR with the tertiary color.